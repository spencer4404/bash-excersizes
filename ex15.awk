BEGIN {
}
{
}
END {
print "Number of Lines in file: " NR
}

