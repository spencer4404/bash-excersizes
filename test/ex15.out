Number of Lines in file: 8
